### ISSUE TITLE
Load model success , but display model texture not well

### ISSUE THREAD

**Issue by LarryMai** [JoeyDeVries/Cell#2]
I compile the source code of cell demo and load nanosuit model.
But I see the display effect is not right as the figure below ...
![screenshot 121](https://cloud.githubusercontent.com/assets/3214309/25469964/309ee6d4-2b52-11e7-9e9e-acf268961428.png)
Nanosuit should be as the figure below
![screenshot 122](https://cloud.githubusercontent.com/assets/3214309/25470035/a4fdb2e4-2b52-11e7-86d7-0bf8bf5eabd5.png)

Anything do I miss?

**Comment by urzq** [comment]
Hello !

I had a hard time configuring your (very cool) engine, and I managed to display the sponza scene. I think I have the same issue as LaryMai, but I cannot figure out what is the problem (I don't have logs nor errors). 

![sponza_bad_textures](https://user-images.githubusercontent.com/522494/34084961-5c29c964-e389-11e7-9700-c3966c7b01cf.png)

Anyway, thanks a lot for your work and your tutorials, they are VERY good :) I'm actively following the lessons, and I'm building my little engine :)

**Comment by JoeyDeVries** [comment]
Thank you! And sorry for the build difficulties (it's not really set up yet for other people configuring it :p).

Did you try the sponza scene that is part of the repository (in build/meshes/sponza)? I made some modifications to that scene so the materials are set up in a way that Cell expects them (for PBR).

**Comment by urzq** [comment]
No, I think the .obj was missing (not versioned on this repo), so I downloaded it (and its textures) from the crytek webpage... It explains why it's not working I believe :) I'll check again, but I believe there is a versionning problem on your side ?

Thanks again !

**Comment by JoeyDeVries** [comment]
You're right, I see it now. The .gitignore is ignoring .obj files (compiled intermediate files) which happen to share the same extension with the WaveFront .obj files. I'll fix this tonight.
